2024/2024-ssl-black-white-hulls-rules.adoc

```diff
@@ -29,23 +29,27 @@ All teams are eligible and *are encouraged* to participate in this challenge.
 
 == Evaluation
 
-Every team has a 5-minute pitch to explain their solution for the challenge and the built design and 5 minutes for possible questions. During the presentation, the TC/OC will measure the time of the team solution.
+Every team has a 5-minute pitch to explain their solution for the challenge and the built design and 5 minutes for possible questions.
+
+After the presentation, the TC/OC will measure the time of the team solution. The TC/OC will decide on the venue how the measurement will occur, but in general it will be measured the time to change the hull of one robot, considering the vision pattern.
```

Comment:
I do not quite understand the last sentence. What does "considering the vision pattern mean"?

Reply:
@JoaoVictor0508 do you meant to consider that robot should still be recognizable?

```suggestion
After the presentation, the TC/OC will measure the time needed for the team solution. The TC/OC will decide on the venue and how the measurement will occur, but in general, the time to change the hull of one robot will be measured. After the hull change, robots must be recognizable by SSL-Vision.
```

Reply:
@lhcavalcanti Yes, that was what I meant to say with this sentence. Thank you.

Reply:
Thanks @JoaoVictor0508. I made some changes to the suggestion above; could you see if it makes sense?

Reply:
It makes sense, thanks @lhcavalcanti